<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件目录树Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="8,1,1">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TreeView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="radioShowAll">
               <property name="text">
                <string>显示目录和文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioShowOnlyDir">
               <property name="text">
                <string>只显示目录</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>379</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxFilter">
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="chkBoxEnableFilter">
                <property name="text">
                 <string>文件过滤</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboFilters">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>*.h;*.cpp;*.ui</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>*.txt</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>*.jpg</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnApplyFilters">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>应用</string>
                </property>
                <property name="icon">
                 <iconset resource="res.qrc">
                  <normaloff>:/icons/828.bmp</normaloff>:/icons/828.bmp</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>447</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>ListView</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>TableView</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labFileName">
        <property name="text">
         <string>文件名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labFileSize">
        <property name="text">
         <string>文件大小：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labType">
        <property name="text">
         <string>节点类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkIsDir">
        <property name="text">
         <string>节点是否是目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labPath">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>路径名：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSetRoot"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actSetRoot">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/fold.bmp</normaloff>:/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>设置根目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
